ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml

```diff
@@ -302,7 +302,8 @@ ScrollView {
             outgoingStatus: model.outgoingStatus
             responseTo: model.responseTo
             authorCurrentMsg: msgDelegate.ListView.section
-            authorPrevMsg: msgDelegate.ListView.previousSection
+            // The previous message is actually the nextSection since we reversed the list order
+            authorPrevMsg: msgDelegate.ListView.nextSection
```

Comment:
should we change the name of the authorPrevMsg to .... authorNextMsg? because right now this assignment looks unintuitive

Reply:
Well, it's still is the previous message, it's just weird in the eyes of the list view, but in the eyes of us and the users, it's still the previous one